tests/training/multi_gpu_test.py

```diff
@@ -4,7 +4,7 @@
 import keras.backend as K
```

Comment:
Oh, I just meant that you're testing the `train_utils.py` functions in a file called `multi_gpu_test.py`.  These should match, generally.